x-pack/plugins/cases/public/components/all_cases/use_cases_columns.tsx

```diff
@@ -357,22 +356,16 @@ export const useCasesColumns = ({
 
   const columns: CasesColumns[] = [];
 
-  if (isSelectorView) {
-    SELECTOR_VIEW_CASES_TABLE_COLUMNS.forEach(({ field }) => {
+  selectedColumns.forEach(({ field, isChecked }) => {
+    if (field in columnsDict && isChecked && casesColumnsConfig[field].canDisplay) {
```

Comment:
This still isn't enough, if `title` is deselected in the user config then it won't show up in the selector view 😬